README.md

```diff
@@ -26,18 +28,31 @@ $ git branch -r | grep krun
 
 Our branches are named `linux-\*-krun`.
 
-### Step 2: Configure and Build the Kernel
+### Step 2: Obtain a Patch of the Krun Changes
 
-Krun insists the kernel is running in "full tickless mode", thus minimising
-regular tick interrupts where possible (for all but the boot CPU core).
+We will not be building directly out of this repo, as we need to use the
```

Comment:
How about making this punchier with something like "On Debian, we cannot build the kernel in this directory directly. Instead we need to take a diff of this kernel's changes and apply those to Debian's kernel. The following command creates a diff:"?